docs/help_with_icg_webform_selfsubmit.md

```diff
@@ -0,0 +1,74 @@
+# Use Case: Islandora Webform for Self-Deposit Workflows
+
+The following is a general overview of what the IW module is capable of and how it is can be used to facilitate self-deposit of electronic theses and dissertations, faculty preprints, or other repository content.
+
+## About the Islandora Webform Module
+
+  * The ICG "Islandora Webform Module" (IW) is a Drupal module developed by Common Media/Born-Digital to add connectivity to an Islandora/Fedora repository.
+  * The IW module is based on the Webform module that is compatible with Drupal 7.
+  * The IW module works on any recent version of Islandora 7x.
+  * The IW module can ingest any kind of object, from PDFs to images, audio, video, manuscripts, and books.
+  * Because the IW module is built on the Drupal Webform module, it inherits all of the latter's functionality. The full functionality can be learned from online documentation and tutorials [see "Documentation" section below].
+  * A working knowledge of MODS, XML Forms, XSLT, and the Fedora object architecture is necessary to set up and manage webforms and workflows.
+
+## Configuring Self-Submit Forms
+
+* Create a new webform (Content > Add Content > Webform)
+
+### Islandora Settings
+Configure how the form interacts with your Islandora collections at webform/islandora. **You must configure these settings before you will be able to map your form components to MODS metadata fields.**
+
+* Select whether to allow the form to show up as a link on certain object pages. The typical self-deposit workflow will likely **not** use this option. 
```

Comment:
This could use some further explanation. If I want to be able to add a link to only certain collections (i.e. our IR collection), would I want to check 'Enable'? I believe so, because David's IW documentation says that I need to manually tag pages on which I want the link to appear.

Reply:
TODO: clarify how to get the URL of the webform and add it as a link elsewhere on the site.